8.15.3 Release notes (#5905)
elastic/security-docs: docs/release-notes/8.15.asciidoc

[[release-notes-header-8.15.0]]
== 8.15

[discrete]
[[release-notes-8.15.3]]
=== 8.15.3

[discrete]
[[known-issue-8.15.3]]
==== Known issues

// tag::known-issue-189676[]
[discrete]
.Tags appear in Elastic AI Assistant's responses
[%collapsible]
====
*Details* +
On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).

====
// end::known-issue-189676[]

[discrete]
[[bug-fixes-8.15.3]]
==== Bug fixes

> * Fixes an {elastic-defend} bug that could cause Linux to crash when {elastic-defend} scans paths (or paths with children) which include virtual file systems, such as procfs.

Reviewer: Sorry I could have worded this better.  Defend crashes, not Linux.  How about something like this?

```suggestion
* Fixes a bug that could cause {elastic-defend} to crash on Linux when scanning paths (or paths with children) which include virtual file systems, such as procfs.
```

[[release-notes-header-8.15.0]]
== 8.15

[discrete]
[[release-notes-8.15.3]]
=== 8.15.3

[discrete]
[[known-issue-8.15.3]]
==== Known issues

// tag::known-issue-189676[]
[discrete]
.Tags appear in Elastic AI Assistant's responses
[%collapsible]
====
*Details* +
On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).

====
// end::known-issue-189676[]

[discrete]
[[bug-fixes-8.15.3]]
==== Bug fixes
* Fixes a bug that could cause {elastic-defend} to crash on Linux when scanning paths (or paths with children) which include virtual file systems, such as procfs.
* Fixes a bug that made alerts wrongfully inherit previously-selected tags ({kibana-pull}194428[#194428]).

> * Fixes a bug that caused all index fields to display when creating a rule filter instead of only fields from the specified index ({kibana-pull}194678[#194678]).

Author: Thanks for following up on this, @vitaliidm! I'll remove it from the 8.15.3 release notes and add it to the 8.15.4 release notes when I draft them.
```suggestion
```

> * Prevents Automatic Import from requesting that LLMs to map to reserved ECS fields ({kibana-pull}195168[#195168]).

Reviewer 2: ```suggestion
* Prevents Automatic Import from requesting that LLMs map to reserved ECS fields ({kibana-pull}195168[#195168]).
```

* Makes Automatic Import more forgiving if LLMs return ECS mappings in unexpected formats ({kibana-pull}195167[#195167]).
* Fixes an Automatic Import bug that prevented non-ECS compatible fields from resolving in structured and unstructured system logs ({kibana-pull}194727[#194727]).
* Fixes an Automatic Import bug that occurred when uploading a new version of an existing integration ({kibana-pull}194298[#194298]).

> * Fixes an Automatic Import bug that is caused integration deployments to fail after you edited the ingest pipeline ({kibana-pull}194203[#194203]).

Reviewer 2: ```suggestion
* Fixes an Automatic Import bug that caused integration deployments to fail after you edited the ingest pipeline ({kibana-pull}194203[#194203]).
```
